### ISSUE TITLE
Locate Moor Web DB

### ISSUE THREAD

**Issue by nisanb** [simolus3/moor#808]
I've been trying to locate the created database ("notes.db") when using the sample project provided by Moor.

I tried debugging, and even tried using sqflite getDatabasePath() method which eventually hangs on the await.

My end goal is to use a shared network drive location for the database, via web, without CORS and internet connection (a policy in a company I develop it for).

Can anyone assist on this matter?

TL;DR: I need to set the web database location to a network drive location path.

Edit: The main reason I need it to be shared through the network drive, is that I need the data to be synced and persistent through multiple client computers - without a web server running to serve requests.

**Comment by simolus3** [comment]
> My end goal is to use a shared network drive location for the database, via web

I don't know if this is possible at all, since (to my knowledge) there isn't a web api to access the native file system. The [Native File System API](https://web.dev/native-file-system/) looks promising, but isn't ready yet.

At the moment, moor web stores the sqlite database file as a blob in IndexedDB or LocalStorage, so it's not backed by an actual file. You can implement a custom `MoorWebStorage` that uses another api if you want to.

**Comment by nisanb** [comment]
So a database, such as SQLite for Python, which is well known, does not exist yet for web ?

**Comment by simolus3** [comment]
Moor uses [sql.js](https://github.com/sql-js/sql.js) for its web implementation, which is a proper SQLite compiled to Wasm and accessible to Dart web apps. There just isn't a way to store data in the local file system yet.
